src/transform.js

```diff
@@ -182,7 +185,7 @@ export default class i18nTransform extends Transform {
                 `new key already mapped to a string: ${entry.key}`
             )
           } else if (conflict === 'value') {
-            this.warn(`Found same keys with different values: ${entry.key}`)
+            this.warn(KEY_REUSE_WARNING + entry.key)
```

Comment:
I believe what we want is a new warning for cases where `duplicate === true` and `conflict === false` which currently is not covered by the code. You should rename `KEY_REUSE_WARNING` to `KEY_CONFLICT_WARNING` and create a new warning for `KEY_REUSE_WARNING`.

Reply:
Understood!  It seems like I ended up targeting only my use case and missed the bigger picture.  I reworked the code to use the `conflict = false` condition and everything seems to be working well, great suggestion.